src/Views/Form-Themes/Sequoia/actions.php

```diff
@@ -0,0 +1,181 @@
+<?php
```

Comment:
@henryholtgeerts This file has been removed. Check latest `feature/form-theme` branch.

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/50b98c62bc9bfdb94a7d540f785af3675a59fb0e